v2/worker.go

```diff
@@ -78,11 +78,7 @@ func (worker *Worker) LaunchAsync(errorsChan chan<- error) {
 			retry, err := broker.StartConsuming(worker.ConsumerTag, worker.Concurrency, worker)
 
 			if retry {
-				if worker.errorHandler != nil {
-					worker.errorHandler(err)
-				} else {
-					log.WARNING.Printf("Broker failed with error: %s", err)
-				}
```

Comment:
The documentation about Error Handling (https://github.com/RichardKnop/machinery#error-handling) suggest that it is used for handling task related error, so I think for `broker.StartConsuming` error, better not use the same error handler, so here I just log it instead.